sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -779,6 +823,10 @@ public void populateDisplayData(DisplayData.Builder builder) {
             .addIfNotDefault(DisplayData.item("codec", codec.toString())
                 .withLabel("Avro Compression Codec"),
                 DEFAULT_CODEC.toString());
+        for (Map.Entry<String, String> entry : metadata.entrySet()) {
+          builder.add(DisplayData.item(entry.getKey(), entry.getValue()).withLabel("Metadata"));
```

Comment:
We're only interested in String metadata but base64 with optional "..." sounds good to me.